### PR TITLE
Multiplier reset warning

### PR DESCRIPTION
**Pull request by Ran73Cohen**
#890

### CODE REVIEW

File: src/components/protection/WithdrawProtectionSingle.vue [hunk 1]
```diff
@@ -188,6 +195,13 @@ export default class WithdrawProtectionSingle extends BaseComponent {
     return pos;
   }
 
+  get rewardsWithMultiplier() {
+    for (let position of vxm.ethBancor.protectedPositions)
+      if (position.rewardsMultiplier.gt(1)) return true;
+
+    return false;
```
Comment (Velua) [inline src/components/protection/WithdrawProtectionSingle.vue:202]: Tad bit verbose, lets just do something like
```suggestion
    return vxm.ethBancor.protectedPositions.some(position => position.rewardsMultiplier.gt(1))
```

File: src/store/modules/rewards/index.ts [hunk 2]
```diff
@@ -241,4 +241,18 @@ export class RewardsModule extends VuexModule.With({
...
+    reserveId
+  }: {
+    poolId: string;
+    reserveId: string;
+  }): Promise<BigNumber> {
+    const result = await this.contract.methods
+      .rewardsMultiplier(this.currentUser, poolId, reserveId)
+      .call();
+
+    return new BigNumber(shrinkToken(result, 6));
```
Comment (Velua) [inline src/store/modules/rewards/index.ts:256]: Why is the decimals hard coded to 6?

**Review by Velua — CHANGES_REQUESTED** [review]

File: src/store/modules/rewards/index.ts (hunk 2 above)
Comment (Ran73Cohen) [reply src/store/modules/rewards/index.ts:256]: Haven't found a decimal returned from the contract
0xB443DEA978B39178Cb05Ae005074227A4390DfCe

**Review by ashachaf — COMMENTED** [review]
please make sure we have 1 box with both warnings and the alert indication for each point.

File: src/store/modules/rewards/index.ts (hunk 2 above)
Comment (Velua) [reply src/store/modules/rewards/index.ts:256]: I don't think we're going to find any decimals from that contract, what token is ```result``` in?

**Comment by ashachaf** [comment]
i still see it as two boxes
image

**Comment by ashachaf** [comment]
@Ran73Cohen 
your part looks good now (task #963 needs to be updated)
image

**Review by ashachaf — APPROVED** [review]

File: src/components/common/AlertBlock.vue [hunk 3]
```diff
@@ -9,7 +9,20 @@
   >
     <div v-if="title" class="font-w700 mb-2">{{ title }}</div>
     <span class="font-w500">
-      <slot name="default">{{ msg }}</slot>
+      <div v-if="messages !== undefined">
+        <div v-for="(message, index) in messages" :key="index" name="default">
```
Comment (Velua) [inline src/components/common/AlertBlock.vue:13]: Can't have key as an index

**Comment by ashachaf** [comment]
implementation is still incorrect.
we should NOT have 2 alert boxes.
top section should be as a tool tip on the "claimable value" text (with an info icon next to it)
@Velua

**Pull request merged**